src/components/Todo.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { FaBars, FaPlus } from 'react-icons/fa';
+import Sidebar from "./SideBar";
+
+class Todo extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            class: 'sidebar-container-passive',
+            title:''
+        }
+        this.sideBarToggler = this.sideBarToggler.bind(this)
```

Comment:
Following clean code practices, Kindly leave a single line space between the two methods.
Furthermore, add comments so that it could be easy for fellow developers to understand the application code.